Formula/livechart

```diff
@@ -0,0 +1,36 @@
+class Livechart < Formula
+  desc "Real-time charting library for Vala and GTK3 based on Cairo"
+  homepage "https://github.com/lcallarec/live-chart"
+  url "https://github.com/lcallarec/live-chart/archive/d364df43bd8336bf7ed461c57a28e97efa7115b1.zip"
+  sha256 "860d57bd633bd1c8c6d387190d8d5ace39fe4a8abb26c0a40bb0c8d263c7a3c0"
+  license "MIT"
+
+  bottle do
+    sha256 arm64_ventura:  "46cd0d4f81a5c2d5efc63b1755a7a002ff95fa8f42007668a337327dd9970607"
+    sha256 arm64_monterey: "9550badfd2988da515c5ff5557014fa3d51a99e4880dd1d20412dc913e57aa2b"
+    sha256 arm64_big_sur:  "0bcf580842dc04a29575b36747ff63ec4456e93dca113be677b2efc610f0d65d"
+    sha256 ventura:        "d40e0ea4a951ce312ea727f83c1127a0c2790574ddb6e7990d0dd93f5eb6baa1"
+    sha256 monterey:       "acfb1a2fe6e8147e6e2e89fadb5890b34d8e6863d040fa39c068c10eb5d21082"
+    sha256 big_sur:        "72ee89c1212e23161ab35c91faea62ee9ce2fef2e533b2269b63c8ffd0d3a1f1"
+    sha256 x86_64_linux:   "09ffb6416daa2e1367119e976d17d0f7a9c486940eb0b1fe6b42b0b8415fbe0d"
+  end
+
+  depends_on "gettext" => :build
+  depends_on "gobject-introspection" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => [:build, :test]
+  depends_on "vala" => :build
+  depends_on "gtk4"
+
+  def install
+    system "meson", "setup", "build", *std_meson_args
+    system "meson", "compile", "-C", "build", "--verbose"
+    system "meson", "install", "-C", "build"
+  end
+
+  test do
+    # include a version check for the pkg-config files
+    assert_match version.to_s, (lib/"pkgconfig/livechart.pc").read

```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.